src/hooks/screen/useCreateChat.ts

```diff
@@ -63,6 +63,7 @@ const useCreateChat = () => {
       const initChannel = await SignedMessageRepo.createSignedChat(members);
       const chatData = createChannelJson(initChannel, selectedUser);
 
+      console.log('initChannel', JSON.stringify(initChannel));
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It's not a good practice to leave `console.log` statements in the production code. It can lead to potential security risks as sensitive information might get logged. Also, it can clutter the console output making it hard to read important logs.
```diff
-       console.log('initChannel', JSON.stringify(initChannel));
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->